src/GafferSceneUIModule/LightEditorBinding.cpp

```diff
@@ -355,6 +366,104 @@ CompoundDataPtr MuteColumn::m_muteUndefinedIconData = new CompoundData(
 
 StringDataPtr MuteColumn::m_muteBlankIconName = new StringData( "muteLightUndefined.png" );
 
+class SetMembershipColumn : public InspectorColumn
+{
+
+	public :
+		IE_CORE_DECLAREMEMBERPTR( SetMembershipColumn )
+
+		SetMembershipColumn( const GafferScene::ScenePlugPtr &scene, const Gaffer::PlugPtr editScope, const IECore::InternedString &setName, const std::string &columnName )
+			: InspectorColumn( new GafferSceneUI::Private::SetMembershipInspector( scene, editScope, setName ), columnName ), m_setName( setName )
+		{
+
+		}
+
+		CellData cellData( const Gaffer::Path &path, const IECore::Canceller *canceller ) const override
+		{
+			CellData result = InspectorColumn::cellData( path, canceller );
+
+			auto scenePath = runTimeCast<const ScenePath>( &path );
+			if( !scenePath )
+			{
+				return result;
+			}
+
+			std::string toolTip;
+			if( auto toolTipData = runTimeCast<const StringData>( result.toolTip ) )
+			{
+				toolTip = toolTipData->readable();
+			}
+
+			if( auto value = runTimeCast<const IntData>( result.value ) )
+			{
+				if( value->readable() & PathMatcher::Result::ExactMatch )
+				{
+					result.icon = m_soloIconData;
+				}
+				else if( value->readable() & PathMatcher::Result::AncestorMatch )
+				{
+					ConstPathMatcherDataPtr soloSetData = scenePath->getScene()->set( m_setName );
```

Comment:
There are a bunch of variables in this class that still refer to `solo`, despite this now being a general-purpose set membership column.

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/5108/commits/6daee8713a5a1d1daefc52552264d8b994061b5c